charts/app/templates/backend/templates/deployment.yaml

```diff
@@ -22,6 +22,7 @@ spec:
       labels:
         {{- include "backend.labels" . | nindent 8 }}
     spec:
+      minReadySeconds: 10
```

Comment:
do we need to add extra 10 seconds delay for pod to be ready to serve incoming traffic .

Reply:
We've had some api errors from fresh pods starting up in test when under load. I have another PR that lightens up the probes and scaling rates from the HPA - but the init container timing is tricky with probes in general. My thinking here was just a bit of extra security, although now that you mention it 10s may be too much. Thoughts?

Reply:
I would recommend we fix the readiness probe, if required, we add startup probes, but once pod is ready, waiting for 10 seconds, sounds not ideal to me

Reply:
I should mention that on the application level, our health check on the api backend is 'dumb' in that it just responds to the http request, not actually if the db is connected and happy, so the k8s probe's source of truth is unreliable, so I'm compensating for that on the infra level. For now.

Reply:
removed these - appreciate the feedback @mishraomp

Reply:
> removed these - appreciate the feedback @mishraomp

Thanks Jon, I would vote for another ticket in the backlog to fix the readiness and liveness probe in the API :)